packages/devtools_server/pubspec.lock

```diff
@@ -243,4 +243,4 @@ packages:
     source: hosted
     version: "0.7.3"
 sdks:
-  dart: ">=2.10.0-110 <=2.11.0-177.0.dev"
+  dart: ">=2.10.0-110 <=2.11.0-161.0.dev"
```

Comment:
Not for this PR, but I'm not sure why we so tightly control which SDKs the server can install into.

Reply:
Me either I was surprised to see this.

Reply:
Ah! This is the lock file - I thought it was the pubspec. The pubspec sdk constraint is what will matter to users - that one looks good.